Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+ruff = "*"
+pycodestyle = "*"
+numpy = "*"
+matplotlib = "*"
+astropy = "*"
+rise = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.11"
```

Comment:
Why is only a single version of Python supported?

Reply:
Pipenv automatically adds the currently used version as the only supported version, and there is no way to make it a range of versions. The simplest solution is to not require any specific version at all, so I will do that.